CesiumGltfReader/src/transformTextureCoords.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+namespace CesiumGltf {
+struct Model;
+}
+
+namespace CesiumGltfReader {
+
+void transformTexture(CesiumGltf::Model& model);
```

Comment:
Add doc. Also name it `transformTextureCoords` to match the file. Or even a more descriptive name (for both the function and file) wouldn't hurt!